app/NavEntry.js

```diff
@@ -13,6 +13,7 @@ import ReportScreen from './views/DR/ReportScreen';
 const Tab = createBottomTabNavigator();
 
 function MainNavigation() {
+  const { t } = useTranslation();
```

Comment:
Why t, the name is not descriptive

Reply:
that is the standard in the project and i18n, t means translate

Reply:
not a good idea, cause it's use in many places and many projects and i18n docs is using it like that